apps/preview/next/pages/showcases/NewsletterBox/NewsletterBox.tsx

```diff
@@ -0,0 +1,99 @@
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { useState } from 'react';
+import { SfButton, SfInput, SfLink, SfIconCheckCircle, SfIconClose } from '@storefront-ui/react';
+
+export default function NewsletterBox() {
+  const [inputValue, setInputValue] = useState('');
+  const [positiveAlert, setPositiveAlert] = useState(false);
+  const [errorAlert, setErrorAlert] = useState(false);
+  const [emailDataBase, setEmailDataBase] = useState<string[]>([]);
```

Comment:
This is cool, however I think there should be info in docs description that if you pass twice same email you can see error alert, its not obvious without reading code

Reply:
sure, I'll add a description for this

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2595/commits/a723fb52c68eaf4658fe01dbada8e0e305892a7d